cmake/external/StanMath.cmake

```diff
@@ -0,0 +1,18 @@
+include(ExternalProject)
+ExternalProject_Add(
```

Comment:
you should fix the commit sha, instead of following the master HEAD. basically set
```GIT_TAG``` and set it to the last stable release of stan

Reply:
I set up the GIT_TAG as required